.editorconfig

```diff
@@ -210,7 +210,7 @@ dotnet_analyzer_diagnostic.category-Style.severity = warning
 dotnet_diagnostic.IDE0004.severity = suggestion
 
 # IDE0005: Remove unnecessary usings/imports
-dotnet_diagnostic.IDE0005.severity = warning
+dotnet_diagnostic.IDE0005.severity = none
```

Comment:
Just curious - why is this needed?

Reply:
> Just curious - why is this needed?

These errors happen when build after I update the SDK.  Same 
 with this commit https://github.com/dotnet/msbuild/pull/9041/commits/53552f7b86037ee993bf42b4e9ab60117cf78f35
_CSC : error InvalidSeverityInAnalyzerConfig: The diagnostic 'ide0005' was given an invalid severity 'warnning' in the analyzer config file at 'D:\WORK\msbuild\.editorconfig'. [D:\WORK\msbuild\src\Samples\PortableTa 
sk\PortableTask.csproj]
CSC : error EnableGenerateDocumentationFile: Set MSBuild property 'GenerateDocumentationFile' to 'true' in project file to enable IDE0005 (Remove unnecessary usings/imports) on build (https://github.com/dotnet/rosl 
yn/issues/41640) [D:\WORK\msbuild\src\Samples\PortableTask\PortableTask.csproj]_